modules/arm_plugin/src/CMakeLists.txt

```diff
@@ -0,0 +1,57 @@
+# Copyright (C) 2020 Intel Corporation
+# SPDX-License-Identifier: Apache-2.0
+#
+
+set(TARGET_NAME "armPlugin")
+
+set(CMAKE_CXX_STANDARD 14)
+
+file(GLOB_RECURSE SOURCES
+    ${CMAKE_CURRENT_SOURCE_DIR}/*.cpp)
+file(GLOB_RECURSE HEADERS
+     ${CMAKE_CURRENT_SOURCE_DIR}/*.hpp)
+
+ie_arm_neon_optimization_flags(neon_flags)
+set_source_files_properties(${SOURCES} PROPERTIES COMPILE_FLAGS "${neon_flags}")
+
+# adds a shared library with plugin
+ie_add_plugin(NAME ${TARGET_NAME}
+              DEVICE_NAME "ARM"
+              SOURCES ${SOURCES} ${HEADERS}
+              SKIP_INSTALL # ATTENTION: uncomment to install component
+              VERSION_DEFINES_FOR arm_plugin.cpp)
+
+target_include_directories(${TARGET_NAME} PRIVATE
+    "${CMAKE_CURRENT_SOURCE_DIR}"
+    "${IE_MAIN_ARM_PLUGIN_SOURCE_DIR}/include")
+
+target_link_libraries(${TARGET_NAME} PRIVATE
+    IE::inference_engine
+    arm_compute::arm_compute
+    arm_compute::arm_compute_core
+    arm_compute::half
+    ${NGRAPH_LIBRARIES}
+    IE::inference_engine_transformations
+    ${INTEL_ITT_LIBS}
```

Comment:
do you use this library directly? I think we don't need it since ITT comes via `IE::inference_engine_plugin_api`

Reply:
removed `${INTEL_ITT_LIBS}`